ansible/es_backup.yml

```diff
@@ -1,4 +1,4 @@
-- hosts: "{{ remote }}"
+- hosts: "{{ groups['composite-search-cluster'][0] }}"
```

Comment:
replace the `groups['composite-search-cluster'][0]` with group_name, and provice `run_once: true`